HW2/src/test/java/com/example/hw2/controller/AuthorControllerTest.java

```diff
@@ -0,0 +1,74 @@
+package com.example.hw2.controller;
+
+import com.example.hw2.servise.AuthorService;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(AuthorController.class)
+@AutoConfigureMockMvc
+class AuthorControllerTest {
+    @Autowired
+    private MockMvc mockMvc;
+    @MockBean
+    private AuthorService service;
+
+    @Test
+    @DisplayName("Test get all endpoint, expect ok")
```

Comment:
(minor) A better way to use description as kind of specification, like one of next:
- "GET /authors... should return first 10 authors"
- "API should return first 10 authors"

*this topic we will cover in next module